src/replay.rs

```diff
@@ -608,3 +652,37 @@ impl<'a> SubTaskRunner<'a> {
         result
     }
 }
+
+// This function injects a [profile.dev] into the given Cargo.toml that
+// disables debuginfo. For now, it will just fail if there already is a
+// [profile.dev] section.
+fn inject_no_debug_into_cargo_toml(cargo_dir: &Path) -> io::Result<()> {
+
+    let cargo_toml_path = cargo_dir.join("Cargo.toml");
+
+    let mut file = try!(OpenOptions::new()
+                                    .read(true)
+                                    .write(true)
+                                    .open(&cargo_toml_path));
+
+    let mut contents = String::new();
+    try!(file.read_to_string(&mut contents));
+
+    if contents.contains("[profile.dev]") {
```

Comment:
this is obviously a hack, since they could do similar things in other ways, right?